src/sentry/uptime/subscriptions/tasks.py

```diff
@@ -95,6 +95,7 @@ def uptime_subscription_to_check_config(
         "timeout_ms": subscription.timeout_ms,
         "request_method": subscription.method,  # type: ignore[typeddict-item]
         "request_headers": headers,
+        "trace_sampling": subscription.trace_sampling,
```

Comment:
You'll have to release and bump the `sentry-kafka-schemas` package to not get mypy errors here I think.

Reply:
it's funny i actually didn't get any mypy errors, but yeah, i called this out in the the PR description.

Reply:
hmmm, why doesn't this produce mypy errors?

Reply:
because of the other ignores added above. i think it makes the ignores apply to the whole statement